repo/packages/C/confluent-connect/3/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "3.0",
+  "name": "confluent-connect",
+  "version": "0.9.6-3.1.1",
+  "scm": "https://github.com/confluentinc/kafka",
+  "description": "Confluent Connect worker",
```

Comment:
Adding link to existing Confluent docs for now; link to dcos examples when complete.